cloud/migrate-to-cloud/hypershift.md

```diff
@@ -0,0 +1,160 @@
+connection is interrupted.
+</highlight>
+
+## Prerequisites
+
+Before you begin, make sure you have:
+
+*   Signed up for your [free Timescale Cloud account][cloud-install].
+*   Installed [Docker][docker-install].
+
+<highlight type="cloud" header="Run all tutorials free" button="Try for free">
+Your Timescale Cloud trial is completely free for you to use for the first
+thirty days. This gives you enough time to complete all the tutorials and run
+a few test projects of your own.
+</highlight>
+
+## Download the hypershift container
+
+<Tabs label="Download hypershift">
+
+<Tab title="MacOS">
+
+<procedure>
+
+### Downloading the hypershift container on MacOS
+
+1.  Open the Docker app on your local machine to start the service.
+
+1.  At the command prompt, pull the latest hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+<Tab title="Linux">
+
+<procedure>
+
+### Downloading the hypershift container on Linux
+
+1.  Start the Docker service:
+
+    ```bash
+    sudo service docker start
+    ```
+
+1.  At the command prompt, pull the latest hypershift container from Dockerhub:
+
+    ```bash
+    docker pull timescale/hypershift
+    ```
+
+</procedure>
+
+</Tab>
+
+</Tabs>
+
+## Run hypershift
+
+In preparation for running hypershift, you need to determine which tables need
+to be converted to hypertables, and which tables need to be compressed during
+the migration. Hypertables must have a unique column labelled `time`.
+
+Hypershift uses a YAML configuration file to determine how to set up your new
+Timescale database. You can create your own file, or use the example file as a
+starting point. You need these details to complete your configuration file:
+
+*   The schema you want to use
+*   The name of the database you are migrating
+*   The name of the `time` column in your database
+*   The chunk time interval to use
+*   The compression policy you want to use
+
+Use this format:
+
+```yml
+- schema: public
+  name: <DATABASE_NAME>
+  time_column_name: <TIME_COLUMN_NAME>
+  chunk_time_interval: "12h"
+  compress:
+    after: "48h"
+    segmentby:
+      - <COLUMN_NAME>
+    orderby:
+      - time desc
+```
+
+<procedure>
+
+### Running hypershift
+
+1.  Open the `hypershift.yml` configuration file, and adjust parameters
+    accordingly. For example:
+
+    ```yml
+    - schema: public
+      name: stocks_real_time
+      time_column_name: time
+      chunk_time_interval: "12h"
+      compress:
+        after: "48h"
+        segmentby:
+          - symbol
+        orderby:
+          - time desc
+    ```
+
+1.  At the command prompt, run the hypershift container. Include the source and
+    destination database connection strings, and the path to your `hypershift.yml`
+    configuration file:
+
+    ```bash
+    docker run -v$(pwd)/hypershift.yml -ti timescale/hypershift:latest clone \
```

Comment:
We don't have a `latest` tag for the docker image. The current release is `0.2`.

```suggestion
    docker run -v$(pwd)/hypershift.yml -ti timescale/hypershift:0.2 clone \
```